eigrpd/eigrp_vty.c

```diff
@@ -333,7 +333,8 @@ DEFPY (clear_ip_eigrp_neighbors,
 		eigrp_hello_send(ei, EIGRP_HELLO_GRACEFUL_SHUTDOWN, NULL);
 
 		/* iterate over all neighbors on eigrp interface */
-		frr_each (eigrp_nbr_hash, &ei->nbr_hash_head, nbr) {
+		while (eigrp_nbr_hash_count(&ei->nbr_hash_head) > 0) {
+			nbr = eigrp_nbr_hash_first(&ei->nbr_hash_head);
 			if (nbr->state != EIGRP_NEIGHBOR_DOWN) {
 				zlog_debug(
```

Comment:
unguarded debug?

Reply:
yep, but not code I touched.   Gonna let that sleeping dog lie for the moment